### ISSUE TITLE
load complete youtube playlists

### ISSUE THREAD

**Issue by select** [select/audius#62]
for now only the first 50 items are added

**Comment by illtellyoulater** [comment]
I agree, this is an heavy limitation, why only load the first 50 tracks?

**Comment by select** [comment]
That was the easiest to implement, did not have to deal with pagination this way

**Comment by illtellyoulater** [comment]
@select I implemented pagination in another project of mine, it's really easy to do: when you receive the first 50 results they also come with a token for obtaining the next results (when available). So you check for presence of this token, and if it's there, then at the end of your function for requesting playlist videos, you call the function again recursively, this time providing the token for the next page of results. This way the function runs until it obtains all playlist videos, and will stop as soon as the token is no longer received.
